protocol/contracts/farm/facets/FieldFacet.sol

```diff
@@ -58,6 +67,7 @@ contract FieldFacet is ReentrancyGuard {
     {
         amount = LibTransfer.burnToken(C.bean(), amount, msg.sender, mode);
         pods = LibDibbler.sow(amount, msg.sender);
+        s.f.beanSown = s.f.beanSown + uint128(amount); // safeMath not needed
```

Comment:
Should Beans Sown in `fund` be included in `beansSown`? They weren't before, but that was because Soil was not decremented. Now, that we have a separate variable to track beansSown, it may make sense to include.

If so, maybe move this statement to `sowNoSoil`?